packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -456,16 +456,24 @@ async function handler(input: CancelBookingInput) {
     updatedBookings.push(updatedBooking);
 
     if (bookingToDelete.payment.some((payment) => payment.paymentOption === "ON_BOOKING")) {
-      await processPaymentRefund({
-        booking: bookingToDelete,
-        teamId,
-      });
+      try {
```

Comment:
Wrapping these payment functions in a try/catch since the booking is already cancelled. If these fail we shouldn't throw an error to the user